src/styles/blocks/catalog.scss

```diff
@@ -1,22 +1,22 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
 
-  max-width: 944px;
+  gap: 48px;
   margin: 0 auto;
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: 199px;
```

Comment:
You can use card-width variable here too. Btw, why is it 199 instead of 200? 
Check this property to calculate element's sizes correctly:
https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing